packages/inventory/src/lang.ts

```diff
@@ -9,6 +9,28 @@ export const VIEW = 'view';
 export const FETCHING_LOCATIONS = 'Fetching locations';
 export const FETCHING_LOCATIONS_DESCRIPTION = 'Please wait, while locations are being fetched';
 export const LOADING_ELLIPSIS = 'Loading ...';
+
+// Service point profile
+export const INVENTORY_ITEMS = 'Inventory items';
+export const EDIT_SERVICE_POINT = 'Edit service point';
+export const ADD_NEW_INVENTORY_ITEM = '+ Add new inventory item';
```

Comment:
I think it's best if we use the plus icon in the component instead of prefixing + in the string